src/tests/ftest/container/attribute.py

```diff
@@ -144,18 +182,21 @@ def test_container_attribute(self):
                 break
         try:
             self.container.container.set_attr(data=attr_dict)
-            # skip step due to DAOS-6880
-            #size, buf = self.container.container.list_attr()
-            #self.verify_list_attr(attr_dict, size, buf)
+
+            # Workaround
+            # Due to DAOS-7093 skip the usage of pydaos cont list attr
+            # size, buf = self.container.container.list_attr()
```

Comment:
Do we need these comments to be there? Looks like we are moving to use daos_cmd.

Reply:
Yes, the intention is to keep the validation complete until pydaos bug (7093) is fixed. Another possibility is pydaos to not be used anymore in this test so we move completely to daos_cmd.

Reply:
skipForTicket pragma should be used.